Submitted By: Jim Gifford <jim at cross-lfs dot org>
Date: 2009-03-01
Initial Package Version: 5.00
Upstream Status: Unknown
Origin: Jim Gifford
Description: For using xz compression

diff -Naur file-5.00.orig/magic/Magdir/compress file-5.00/magic/Magdir/compress
--- file-5.00.orig/magic/Magdir/compress	2008-11-26 06:14:00.000000000 -0800
+++ file-5.00/magic/Magdir/compress	2009-03-01 23:31:49.670919682 -0800
@@ -188,6 +188,19 @@
 >6	byte		x			version %d
 >7	byte		x			\b.%d
 
+# XZ (Lempel-Ziv-Markov chain-Algorithm) file format supported by:
+#   - 7-Zip <http://7-zip.org/>
+#   - LZMA SDK <http://7-zip.org/sdk.html>
+#   - LZMA Utils <http://tukaani.org/lzma/>
+#
+# Note that this is different format than the original format created
+# by LZMA_Alone from older versions of LZMA SDK. These files are
+# non-trivial to detect, because they have no magic bytes.
+#
+0	ustring		\xFD7zXZ\x00	xz compressed data,
+>6	ubyte&0x10	0x00		single-block stream
+>6	ubyte&0x10	0x10		multi-block stream
+
 # Type: LZMA
 # URL:  http://www.7-zip.org/sdk.html
 # From: Robert Millan <rmh@aybabtu.com> and Reuben Thomas <rrt@sc3d.org>
